ext/include/opentelemetry/ext/http/client/curl/http_client_curl.h

```diff
@@ -58,12 +58,20 @@ class Request : public http_client::Request
     timeout_ms_ = timeout_ms;
   }
 
+  void IncludePermissionsFilePath(std::string filepath) noexcept override 
```

Comment:
Other methods here uses `nostd::string_view` for parameters instead of `std::string`, please make it consistent.